style-guide.rst

```diff
@@ -520,32 +543,72 @@ A big advantage of reST in comparison to plain Markdown is that it allows to reu
 Substitution
 ~~~~~~~~~~~~
 
-To reuse sentences or paragraphs without too much markup and special formatting, use substitutions.
+To reuse sentences and entire paragraphs
+that have little markup or special formatting,
+define their `substitutions`_ in two possible ways.
```

Comment:
```suggestion
define `substitutions`_ for them in two possible ways.
```
I don't want to substitute the text, but I want to use something (which RST calls substitutions) that is replaced with the text.